examples/platform/attacks_detected_stopped_export.py

```diff
@@ -0,0 +1,144 @@
+
+    now = datetime.now().strftime('%Y%m%d%H%M%S')
+    sep = "|"
+
+    alerts_stopped = cb.select(CBAnalyticsAlert).set_policy_applied(['APPLIED'])
+    alerts_detected = cb.select(CBAnalyticsAlert).set_policy_applied(['NOT_APPLIED'])
+
+    if args.severity:
+        alerts_stopped = alerts_stopped.set_minimum_severity(args.severity)
+        alerts_detected = alerts_detected.set_minimum_severity(args.severity)
+
+    if args.range:
+        alerts_stopped = alerts_stopped.set_time_range('last_update_time', range=f'-{args.range}')
+        alerts_detected = alerts_detected.set_time_range('last_update_time', range=f'-{args.range}')
+
+    try:
+        if len(alerts_stopped) > 0:
+            print(f'Writing Attacks Stopped to: attacks_stopped_{now}.csv')
+            with open(f'attacks_stopped_{now}.csv', 'w') as stopped:
+                stopped.write('Alert ID,Alert,ThreatScore,TargetPriority,Vector,Stage,First Seen,DeviceHostName,'
+                              'Username,Policy,On/Off Premise,TTPs,ThreatCategory,RunState,PolicyAppliedState,'
+                              'Dismissed,Reputation\n')
+                for alert in alerts_stopped:
+                    stopped.write(f'{alert.id},')  # Alert ID
+                    stopped.write(f'{alert.reason},')  # Alert
+                    stopped.write(f'{alert.severity},')  # ThreatScore
+                    stopped.write(f'{alert.target_value},')  # TargetPriority
+                    stopped.write(f'{alert.threat_cause_vector},')  # Vector
+                    stopped.write(f'{sep.join(alert.kill_chain_status)},')  # Stage
+                    stopped.write(f'{alert.first_event_time},')  # First Seen
+                    stopped.write(f'{alert.device_name},')  # DeviceHostName
+                    stopped.write(f'{alert.device_username},')  # Username
+                    stopped.write(f'{alert.policy_name},')  # Policy
+                    stopped.write(f'{alert.device_location},')  # On/Off Premise
+                    ttps = set()
+                    if alert.threat_indicators:
+                        for indicator in alert.threat_indicators:
+                            if indicator["ttps"]:
+                                for ttp in indicator["ttps"]:
+                                    ttps.add(ttp)
+                    stopped.write(f'{sep.join(list(ttps))},')  # TTPs
+                    stopped.write(f'{alert.threat_cause_threat_category},')  # ThreatCategory
+                    stopped.write(f'{alert.run_state},')  # RunState
+                    stopped.write(f'{alert.policy_applied},')  # PolicyAppliedState
+                    stopped.write(f'{False if alert.workflow["state"] else True},')  # Dismissed
+                    stopped.write(f'{alert.threat_cause_reputation},')  # Reputation
+                    stopped.write('\n')
+
+        if len(alerts_detected) > 0:
+            print(f'Writing Attacks Stopped to: attacks_detected_{now}.csv')
```

Comment:
Should be "Writing Attacks Detected..." for consistency.

Reply:
Thanks good catch